pkg/registry/server/client_test.go

```diff
@@ -0,0 +1,326 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"k8s.io/utils/pointer"
+
+	"github.com/openshift/ci-tools/pkg/api"
+)
+
+var parsedConfig = &api.ReleaseBuildConfiguration{
```

Comment:
Why is this a global (and not a `testdata/` entry?)

Reply:
This existed before, I just moved it. I guess I can modernize it.

Reply:
Addressed